src/ens/resolvers/index.ts

```diff
@@ -0,0 +1,57 @@
+import { GraphQLScalarType, Kind } from 'graphql';
+import * as _ from 'lodash';
+import { EthqlAccount, EthqlBlock, EthqlTransaction } from '../../core/model';
+import blockResolvers from '../../core/resolvers/block';
+import rootResolvers from '../../core/resolvers/root';
+
+import Web3 = require('web3');
+import { EthqlContext } from '../../context';
+
+function isEnsDomain(input: string) {
+  return input.includes('.eth');
+}
+
+// tslint:disable-next-line
+const Address = new GraphQLScalarType({
```

Comment:
We should separate the `Address` scalar type from the rest into different files, e.g. `scalars.ts` and `resolvers.ts` (look at how it's done in core).